src/Analysis/Ast/Impl/Modules/Resolution/MainModuleResolution.cs

```diff
@@ -99,7 +99,12 @@ public MainModuleResolution(string root, IServiceContainer services)
                     return module;
                 }
             }
-            
+
+            var dbs = _services.GetService<IModuleDatabaseService>();
+            if (dbs != null && dbs.TryCreateModule(name, moduleImport.ModulePath, out var m) != ModuleStorageState.DoesNotExist && m != null) {
```

Comment:
So we are still using this module for `partial`?

Reply:
`Partial` may or may not happen. The idea was to use Python process to obtain initial info on the module members quickly rather than wait for analysis. Consider `plotly`, for example. So we get quick members from Python, then, when/if analysis completes (user may as well close the app before it finishes), write the actual info. However, I am not 100% confident we'll do it, but we might still.